lib/routes/account.js

```diff
@@ -382,9 +436,7 @@ module.exports = function (
         }
 
         function sendNewDeviceLoginNotification () {
-          if (config.newLoginNotificationEnabled && requestHelper.wantsKeys(request)) {
-            // The response doesn't have to wait for this,
-            // so we don't return the promise.
```

Comment:
Please keep the comment about how we purposely don't wait for the promise (unless you want to start waiting for the promise, e.g. to simplify tests)